src/php/proxy.php

```diff
@@ -1,10 +1,23 @@
 <?php
 
+$allowed_urls = array(
+ 'https://www.trumba.com/calendars/robert-woodruff-library.json',
+ 'https://www.trumba.com/calendars/robert-woodruff-library-classes-at-woodruff-library.json',
+ 'https://www.trumba.com/calendars/LibrarySigns.json',
+ 'https://www.trumba.com/calendars/',
+ 'https://scholarblogs.emory.edu/woodruff/feed',
+ 'https://scholarblogs.emory.edu/woodruff/topics/news/',
+ 'https://scholarblogs.emory.edu/woodruff/topics/news/feed',
+ 'http://rssmix.com/u/10271671/rss.xml',
+ 'http://rssmix.com/u/10788212/rss.xml',
+ 'https://rssmix.com/u/10271671/rss.xml',
+);
```

Comment:
@elliotkendall the issue with this solution is that every time we enter a feed into cascade we would need to update an allowed list of URLs and redeploy to the server, that is unsustainable for content editors who are not developers and do not have the skills or the ability to access these assets.  where as #388 lets you enter any URL so long as it is formatted as a URL.

Is it possible to come to a compromise and identify an allowed list of domains that we could validate against. So the only options would be anything at: 

```
https://www.trumba.com/*, 
https://scholarblogs.emory.edu/*
https://rssmix.com/*
```
Will be parsed?